microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-studio.adoc

```diff
@@ -0,0 +1,160 @@
+
+* *HTTP Listener*, which initiates a flow each time it receives a request on the configured host and port
+* *Scheduler*, which initiates a flow when a time-based condition is met
+
+For example, to configure *HTTP Listener*, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Microsoft .NET, follow these steps:
+
+. In the *Mule Palette* view, select *Microsoft DotNet* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Microsoft .NET system.
+
+To configure the global element for .Net Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, complete these fields:
++
+image::microsoft-dotnet-connection.png[Studio General tab]
++
+* *Connection*: Select a connection type.
+* *Scope*: Select the connection scope:
++
+** *Singleton*: Shares the same instance of the Microsoft .NET component across multiple calls. Use this option to store a shared state in class instance members across different calls.
+** *Transient*: Creates a new instance of the Microsoft .NET component per request.
+* *Grant Full Trust to the .NET assembly*: Microsoft .NET code can be executed in an application domain with restricted privileges to avoid the execution of malware code that can affect the stability of Mule Runtime. These restrictions include limited access to the file system, native code execution, network calls, registry access, and more. By default the Microsoft .NET connector is set to use full trust.
```

Comment:
```suggestion
* *Grant Full Trust to the .NET assembly*
+
Microsoft .NET code can be executed in an application domain with restricted privileges to avoid the execution of malware code that can affect the stability of Mule runtime engine. These restrictions include limited access to the file system, native code execution, network calls, registry access, and more. By default, Microsoft .NET Connector is set to use full trust.
```
Also, the first sentence of this description is passive; can it be reworded to active voice and also clarify whether the actor or the domain has "restricted privileges"?
And Mule Runtime > Mule runtime engine or Mule or Mule instance